docs/meetings.md

```diff
@@ -0,0 +1,51 @@
+
+Meetings are safe space. There are no dumb ideas or stupid questions - anyone can ask if they don’t understand something.
+
+:::tip
+If there is a strong decision needed, let people provide feedback also offline (either before or after the meeting). Some people might prefer written communication and/or asynchronous communication. Allow them to participate so the best decision is chosen.
+:::
+
+## Refinement meeting
+
+Weekly meetings to prepare our next cards to work on. This is not to **_plan_** the work for the next week as done in Scrum. Kanban is based on a stream of cards going through the board columns. This activity is about moving the card from the top of the backlog (a priority one in our case) into the _refined_ column so they can be taken by anyone to start the real work.
+
+When refining, we need to make sure that the card is clearly defined (i.e. **anyone** can work on the card), there is a clear definition of done and the card is doable in 5 or fewer days.
+
+The discussion starts with a quick introduction of the card (done by a [Kanban lead](TBD) or a person the most familiar with the task). The card is being updated and the discussion continues until we agree it's clear and can be done in 5 days or less -- to help ourselves reach this agreement, there is a voting going on. We used to do a thumbs-up/down for the following questions:
+
+1. Is this card clear?
+2. Is this doable in 5 days or less?
+
+Since there wasn't much disagreement and discussion, we switched to Scrum-like voting by hand about story points. (A difference in story points helps us better find a difference in the understanding.) We collect the story points but it's not the main reason why we do the voting. Be respectful when asking people why they've chosen a different story point number. It's to help everyone with the understanding and also to share the possible risks, not to put anyone in a spot!
+
+Mark cards worth demoing with a `demo` label – either for the public or for the team to share knowledge.
+
+At the end of the meeting, make sure the order (=priority) of the cards in the _refined_ column is right and respects our priorities as shown on the [epic board](https://github.com/orgs/packit/projects/7/views/29).
+
+## Retrospectives
+
+Bi-weekly meetings to get a sense of how everyone feels. This is a safe space -- feel free to share anything openly and also be open to listening to others.
+
+Miro board is created by the Kanban Lead leading the Retrospective at the beginning of the week (ideally Monday morning) to provide time for adding notes to the board in advance.
+There's a default Miro board template prepared, but any activity or board can be used to make it more interesting.
+
+Action items from the previous retro board should be included to be discussed and checked.
```

Comment:
could you please include that we also check how we are standing on our planned epics for the quarter?